src/components/People.tsx

```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
+import { Loader } from './Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from './PersonLink';
+
+export const People = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  const getPeopleData = async () => {
+    setIsLoading(true);
+    try {
+      const peopleData = await getPeople();
+
+      setPeople(peopleData);
+    } catch {
+      setError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const getFaterSlug = (person: Person) => {
+    const searchedGuy = people.find(
+      somebody => somebody.name === person.fatherName,
+    );
+
+    return searchedGuy?.slug || '';
+  };
+
+  const getMotherSlug = (person: Person) => {
+    const searchedWoman = people.find(
+      somebody => somebody.name === person.motherName,
+    );
+
+    return searchedWoman?.slug || '';
+  };
+
+  useEffect(() => {
+    getPeopleData();
+  }, []);
+
+  const { selected } = useParams<{ selected: string }>();
+  const selectedPerson = selected || '';
+
+  return (
+    <div className="container">
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {error && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {!isLoading && people.length === 0 && (
```

Comment:
Create variable for `!isLoading && !people.length`

Reply:
Sure thing! But why?

Reply:
If you have 2 and more conditions, better to create variable for that, to make code more clear and easier for understanding